po/nl.po

```diff
@@ -609,11 +609,11 @@ msgstr "Totaal aantal geïnstalleerde pakketten: {}"
 
 #: src/stats.rs:87
 msgid "Aur packages: {}"
-msgstr "Aur pakketten: {}"
+msgstr "AUR-pakketten: {}"
```

Comment:
Is there a language reason for adding the hyphen?

Reply:
Yes, writing compounds as two words is actually a very common mistake in the Netherlands, but is definitely wrong. In Dutch, compound words are by default written as one word. When pronunciation becomes unclear as one word, or when using loan words or abbreviations, the compound is connected using a hyphen. This is an example of the latter.